apps/backend/src/middlewares/rateLimiter.ts

```diff
@@ -1,6 +1,6 @@
 import { NextFunction, Request, Response } from "express";
 import { SubscriberDetail } from "../interfaces";
-import { prisma, split_auth_header } from "../lib/utils";
+import {  split_auth_header } from "../lib/utils";
 export const rateLimiter = async (
```

Comment:
logic here should be: 
1. Check if the `<subscriber_id>-count` is present in redis or not.
 a. if not present then add a key `<subscriber_id>-count` with a value of 1 (number) and invoke `next()`
 b. if present then check if the value is less than the `RATE_LIMIT_24HR` env variable -> increment and allow if check passes otherwise return error.